docs/developer/api.rst

```diff
@@ -34,6 +34,17 @@ Indexing Strategies
 
 GeoMesa uses several different strategies to index simple features. In the code, these strategies are abstracted as 'tables'. For details on how GeoMesa encodes and indexes data, see tables. For details on how GeoMesa chooses and executes queries, see the ``org.locationtech.geomesa.accumulo.index.QueryPlanner`` and ``org.locationtech.geomesa.accumulo.index.QueryStrategyDecider`` classes.
 
+Explaining:  Query Plans
+++++++++++++++++++++++++
+
+Given a data store and a query, you can ask GeoMesa to explain its plan for how to execute the query:
+
+.. code-block:: java
+
+    dataStore.explainQuery(query)
```

Comment:
mmm, this method got removed actually... the current way to do it is:
```
dataStore.getQueryPlan(query, explainer = new ExplainPrintln)
```
or you can use `new ExplainString` or `new ExplainLogging`.